api/api/services/s3.py

```diff
@@ -10,7 +10,7 @@
 class S3Bucket:
     def __init__(self) -> None:
         self.client = Minio(
-            "s3:9000",
+            "localhost:9000", #TOFIX
```

Comment:
this should be an env value, i think you can pull `fix/s3-image-upload` in order to have the correct code

Reply:
done